boards/arm/efm32pg_stk3402a/efm32pg_stk3402a_defconfig

```diff
@@ -19,3 +19,10 @@ CONFIG_I2C=y
 CONFIG_REBOOT=y
 CONFIG_COUNTER=y
 CONFIG_NOTIFY=y
+CONFIG_LED=y
+CONFIG_LED_PWM=y
+CONFIG_LED_SHELL=y
+CONFIG_PWM=y
+CONFIG_PWM_SHELL=y
+CONFIG_PWM_GECKO=y
+CONFIG_PERSISTENT_GECKO_RTCC=y
```

Comment:
Do we really want to add the shells to compile by default with the PG starter kit?

Reply:
For the testing I am doing now, I'd like to have those configs, but they will likely be removed before I push it upstream